multitenancy/demo.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+source <(curl -s -L https://raw.githubusercontent.com/paxtonhare/demo-magic/master/demo-magic.sh)
+
+{
+    cat *.yaml | kubectl delete --ignore-not-found -f -
+    kubectl delete ns test --ignore-not-found
+} 1> /dev/null 2>&1
+
+
+pe "cat *.yaml | kubectl apply -f -"
```

Comment:
I forgot to mention, but to add the policy which creates cluster-wide roles Kyverno needs additional permissions. I typically add the permissions as part of the demo, as its a good item to showcase.

Reply:
Is it okay if we skip this for the presentation on Wednesday given that the timing is tight. I have set up the cluster to have appropriate permissions for now, but I can mention it during the demo that the generate controller will need appropriate permissions and that can be customized through the helm chart.

Reply:
Added an extra RBAC file.